app/controllers/coronavirus_pages_controller.rb

```diff
@@ -65,13 +65,9 @@ def publish_page
   end
 
   def fetch_content_and_push
-    response = RestClient.get(page_config[:raw_content_url])
-
-    if response.code == 200
-      corona_content = YAML.safe_load(response.body)["content"]
-
-      if valid_content?(corona_content, page_type)
-        presenter = CoronavirusPagePresenter.new(corona_content, page_config[:base_path])
+    if details_builder.data && details_builder.success?
```

Comment:
Is it worth encapsulating this check in `details_builder` in entirety?

Reply:
This was the root of a number of discussions between @hannako and I. In the final form, this code has moved to the `DraftUpdater` where it looks like this:

```ruby
    def payload
      if content_builder.success?
        CoronavirusPagePresenter.new(content_builder.data, base_path)
      else
        raise DraftUpdaterError, content_builder.errors.to_sentence
      end
    end
```
Note that `DetailsBuilder` has been renamed to `ContentBuilder` in the final version.